### PR TITLE
320: Add x to filter params

### PR DESCRIPTION
**Pull request by HellerCommaA**
for https://github.com/mcmonkeyprojects/SwarmUI/issues/320

### CODE REVIEW

**Comment by HellerCommaA** [comment]
Screenshot 2024-10-14 at 8 26 33 PM

**Comment by mcmonkey4eva** [comment]
image
uhhh


also an `x` to clear should only show up if there's content to be cleared, not while empty

File: src/wwwroot/js/genpage/main.js [hunk 1]
```diff
@@ -2292,6 +2292,17 @@ function storeImageToHistoryWithCurrentParams(img) {
     });
 }
 
+function clearInput() {
```
Comment (mcmonkey4eva) [inline src/wwwroot/js/genpage/main.js:2295]: should rename to eg `clearParamFilterInput` ('clearInput' is a little too vague for global namespace)

**Comment by HellerCommaA** [comment]
closing for now, don't feel like fighting css.

**Comment by AshtakaOOf** [comment]
Turn it into a Draft?

**Pull request closed without merging**